bpf/init.sh

```diff
@@ -519,11 +519,9 @@ if [ "$HOSTLB" = "true" ]; then
 	COPTS="-DLB_L3 -DLB_L4"
 	if [ "$IP6_HOST" != "<nil>" ]; then
 		bpf_load_cgroups "$COPTS" bpf_sock.c bpf_sock.o sockaddr connect6 from-sock6 $CALLS_MAP $CGROUP_ROOT $BPFFS_ROOT
-		bpf_load_cgroups "$COPTS" bpf_sock.c bpf_sock.o sockaddr sendmsg6 from-sock6 $CALLS_MAP $CGROUP_ROOT $BPFFS_ROOT
```

Comment:
I think it's still possible to use `sendmsg{4,6}()` for TCP connections?

Reply:
Not sure about this, but for TCP to be loadbalanced it's enough to attach to `connect{4,6}`. I believe that the `sendmsg{4,6}` attachment in `init.sh` was introduced because of UDP.

Reply:
I don't know how common it is, I just noticed that when I google `sendmsg() tcp` I could find examples ;-)